app/onboarding/health.tsx

```
@@ -6,53 +6,50 @@ import { useHealthData } from '@/hooks/useHealthData';
 
         if (Platform.OS === 'ios') {
             await initHealthKit();
-            setHasPermissions(true);
-            await finishOnboarding();
+            setHasHealthPermissions(true);
+            await finishHealthOnboarding();
         }
 
         if (Platform.OS === 'android') {
             ToastAndroid.show('Please allow all...', ToastAndroid.SHORT);
             const permissions = await healthConnect!.requestPermission(REQUIRED_PERMISSIONS);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)